app/src/componentLibrary/systemMessage/systemMessage.jsx

```diff
@@ -24,7 +24,11 @@ const MODE_SYS_ERROR = 'error';
 
 export const SystemMessage = ({ header, caption, children, mode, widthByContent }) => {
   return (
-    <div className={cx('system-message', { 'content-width': widthByContent })}>
+    <div
+      className={cx('system-message', {
+        'content-width': widthByContent,
+      })}
+    >
```

Comment:
Please check your formatting. Looks like you didn't touch this file but formatting is changed